docs/code-style-guide.md

```diff
@@ -75,6 +75,16 @@ If you want to know more about our coding style you can take a look at our [clan
   const float SCALE_FACTOR = 2.15;
   float distance = catch_distance * SCALE_FACTOR;
   ```
+* Avoid initializing multiple variables on the same line.
```

Comment:
can we add a quick explanation of why this is bad here? (ie `(in the below example, only "z" will actually be set)`

Reply:
this was moved from misc section... but I can fix it

Reply:
Great, thanks!

Reply:
Is this resolved?

Reply:
Sorry, I don't see any explanation added here?

Reply:
I tagged you in another comment explaining below